p2p/pex/pex_reactor.go

```diff
@@ -531,6 +533,10 @@ func (r *PEXReactor) dialPeer(addr *p2p.NetAddress) error {
 	if attempts > 0 {
 		jitterSeconds := time.Duration(cmn.RandFloat64() * float64(time.Second)) // 1s == (1e9 ns)
 		backoffDuration := jitterSeconds + ((1 << uint(attempts)) * time.Second)
+		if r.Switch.IsPeerPersistent(addr) && r.config.PersistentPeersMaxDialPeriod > 0 &&
```

Comment:
```
		if r.config.PersistentPeersMaxDialPeriod > 0 &&
			backoffDuration > r.config.PersistentPeersMaxDialPeriod && r.Switch.IsPeerPersistent(addr) {
```

IsPeerPersistent check should be last since it's more CPU expensive

Reply:
Oh, sorry for late response, It's already merged, Could I additional PR to resolve these feedback?

Reply:
fixed in https://github.com/tendermint/tendermint/pull/4218